acceptance/testdata/launcher/Dockerfile.windows

```
@@ -1,13 +1,13 @@
-FROM golang:1.15-nanoserver-1809
+FROM golang:1.16-nanoserver-1809
 
 COPY exec.d/ /go/src/exec.d
-RUN go build -o helper.exe exec.d
+RUN set GO111MODULE=off; go build -o helper.exe exec.d
 
```

```suggestion
WORKDIR /go/src
RUN go build -o helper.exe exec.d
```

Like micahyoung said, something is up with the `;` there. It's not necessary anyway because GO111MODULE=off by default on golang:1.16. And because it's off, you can build by first switching to the directory (_as opposed to relying on the c:/gopath_)

---

Thanks, that makes sense. Let's see if it works 🤞